examples/simple/main_voting.c

```diff
@@ -151,11 +151,6 @@ bool simulate_random_votes(uint32_t encrypter_ix, uint64_t num_ballots)
         bool selections[MAX_SELECTIONS];
 
         fill_random_ballot(selections);
-        if (!Validate_selections(selections, NUM_SELECTIONS))
-        {
-            printf("oops");
-            continue;
-        }
```

Comment:
Removed extra validate call from here since we'll be calling it anyway in Voting_Encrypter_encrypt_ballot 10 lines below here.